public/app/features/dashboard-scene/settings/utils.ts

```diff
@@ -76,7 +76,7 @@ export function createDashboardEditViewFor(editview: string): DashboardEditView
     case 'links':
       return new DashboardLinksEditView({});
     case 'versions':
-      return new VersionsEditView({});
+      return new VersionsEditView({ versions: [] });
```

Comment:
Nit: Can we make the state optional? Fetching happens on activations anyways and I don't think we will need this to be used as a standalone object outside of the settings context.